code/game/objects/items/stunbaton.dm

```diff
@@ -281,6 +281,8 @@
 	desc = "An improvised stun baton."
 	icon_state = "stunprod"
 	inhand_icon_state = "prod"
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'

```

Comment:
don't need this since it's already flagged as a backslot item.

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.